{{cookiecutter.repo_name}}/.gitignore

```diff
@@ -61,6 +61,7 @@ output/*/index.html
 docs/_build
 docs/modules.rst
 docs/{{ cookiecutter.app_name }}.rst
+docs/docs/{{ cookiecutter.app_name }}.*.rst
```

Comment:
Why would anything get created under ``docs/docs``?  That looks like there was some kind of mistake in the Sphinx configuration